pyproject.toml

```
@@ -64,17 +64,17 @@ django-appconf = "~1.0"
 # libpq-dev						# From package repository.
 
 # Monitoring
-raven = "~6.10"
 
 django-js-reverse = "~0.9"
 
 psycopg2-binary = { version = "~2.8", optional = true }
 uwsgi = { version = "~2.0", optional = true }
-redis = { version = "~3.3", optional = true }
+redis = "~3.3"
 
 # Docs
 sphinx = { version = "*", optional = true }
 sphinx-rtd-theme = { version = "*", optional = true }
+sentry-sdk = "^0.14.3"
```

Since we are using the integrations, I think it would be a little clearer if we show that here, so that if e.g. `sentry-sdk` has a new version of `celery` it doesn't support, we would become aware of it
They seem to not have an extra for redis 🤔 
```suggestion
sentry-sdk = { version = "^0.14.3", extras = [ "django", "celery" ] }
```

---

I'm not sure I fully understand the extras option, but I guess this is better than what i originally had. Will take a look at it.

---

It’s like saying “we have this extra feature” but that requires this additional dependency, and that dependency might have a specified version, which can lead to compatibility issues, which is what this could potentially guard us against